test/integration/full/preload-cssom/preload-cssom.js

```diff
@@ -0,0 +1,130 @@
+/* global axe, sinon, Promise */
+describe('preload cssom integration test pass', function() {
+	'use strict';
+
+	before(function(done) {
+		function start() {
+			// Stop messing with my tests Mocha!
+			document.querySelector('#mocha h1').outerHTML =
+				'<h2>preload cssom integration test</h2>';
```

Comment:
You probably don't need that :)

Reply:
I do, otherwise there is no title generated, this seems be a problem in all tests under integration/full

Reply:
Yup, that is right. This hack is spread across the code base. I have only borrowed it.

Reply:
It isn't. I put this in the page-has-heading-one tests to make sure I can test that page without mocha getting in the way. That's the only place its needed.